sunbird_service_api_test/src/test/resources/templates/user/update/testUpdateUserFrameworkSuccess/request.json

```diff
@@ -0,0 +1,11 @@
+{
+"request":{
+"userId":"${userId}",
+"framework" : {
+	"medium"  : ["English"],
```

Comment:
Can we get the valid values from env variables as they can change from one framework to another?